sros2/test/sros2/commands/security/verbs/test_generate_policy.py

```diff
@@ -12,133 +13,155 @@
-        )
-
-        try:
-            # Create a server and client
-            node.create_client(Empty, f'{test_name}_client')
-            node.create_service(Empty, f'{test_name}_server', lambda request,
-                                response: response)
-
-            # Generate the policy for the running node
-            assert cli.main(
-                argv=['security', 'generate_policy', os.path.join(tmpdir, 'test-policy.xml')]) == 0
-        finally:
-            node.destroy_node()
-            rclpy.shutdown(context=context)
-
-        # Load the policy and pull out allowed replies and requests
-        policy = load_policy(os.path.join(tmpdir, 'test-policy.xml'))
-        profile = policy.find(
-            path=f'enclaves/enclave[@path="{test_enclave}"]'
-                 + f'/profiles/profile[@ns="{test_node_namespace}"]'
-                 + f'[@node="{test_node_name}"]'
-        )
-        assert profile is not None
-        service_reply_allowed = profile.find(path='services[@reply="ALLOW"]')
-        assert service_reply_allowed is not None
-        service_request_allowed = profile.find(path='services[@request="ALLOW"]')
-        assert service_request_allowed is not None
-
-        # Verify that the allowed replies include service_server and not service_client
-        services = service_reply_allowed.findall('service')
-        assert len([s for s in services if s.text == f'{test_name}_server']) == 1
-        assert len([s for s in services if s.text == f'{test_name}_client']) == 0
-
-        # Verify that the allowed requests include service_client and not service_server
-        services = service_request_allowed.findall('service')
-        assert len([s for s in services if s.text == f'{test_name}_client']) == 1
-        assert len([s for s in services if s.text == f'{test_name}_server']) == 0
-
-
-# TODO(jacobperron): On Windows, this test is flakey due to nodes left-over from tests in
-#                    other packages.
-#                    See: https://github.com/ros2/sros2/issues/143
-@pytest.mark.skipif(
-    'nt' == os.name, reason='flakey due to nodes left-over from tests in other packages')
-def test_generate_policy_no_nodes(capsys):
-    with tempfile.TemporaryDirectory() as tmpdir:
-        assert cli.main(argv=[
-            'security', 'generate_policy', os.path.join(tmpdir, 'test-policy.xml')]) != 0
-        stderr = capsys.readouterr().err.strip()
-        assert stderr == 'No nodes detected in the ROS graph. No policy file was generated.'
-
-
-def test_generate_policy_no_policy_file(capsys):
-    with pytest.raises(SystemExit) as e:
-        cli.main(argv=['security', 'generate_policy'])
-        assert e.value.code != 0
-    stderr = capsys.readouterr().err.strip()
-    assert 'following arguments are required: POLICY_FILE_PATH' in stderr
+
+from utilities.sros2_cli_test_case import generate_sros2_cli_test_description
+from utilities.sros2_cli_test_case import SROS2CLITestCase
```

Comment:
Should these "utilities" imports be prefixed with a '.' or using a full name ?

Reply:
A '.' prefix suggests that the test lives in a Python module/package which is not the case. I'm not sure what you mean by full name. 

I can (and probably should) ensure the sibling `utilities` module is used by tweaking `sys.path`.

Reply:
See https://github.com/ros2/sros2/pull/214/commits/30b0fe341e643fa671b45a79cd6ab4dea4a24d31.

Reply:
:+1: